src/cmd/remove.nim

```diff
@@ -27,10 +42,19 @@ proc paxRemove*(name: string): void =
   echoMod(cfMod, moreInfo = true)
   echo ""
 
-  returnIfNot promptYN("Are you sure you want to remove this mod?", default = true)
-  
-  echoInfo "Removing ", cfMod.name.cyanFg, ".."
-  manifest.removeMod(cfMod.projectId)
+  returnIfNot promptYN("Are you sure you want to remove this mod?",
+      default = true)
+
+  var dependents = manifest.isDepended(cfMod.projectId)
+  if len(dependents) > 0:
+    echoRoot "Cannot remove ", cfMod.name.cyanFg, " it is a dependent of:"
```

Comment:
```suggestion
    echoRoot "Cannot remove ", cfMod.name.cyanFg, " - mod is needed by"
```
or something like that, so that the average user can understand it.